synapseclient/table.py

```diff
@@ -813,6 +813,7 @@ def Table(schema, values, **kwargs):
       - a list of lists (or tuples) where each element is a row
       - a string holding the path to a CSV file
       - a Pandas `DataFrame <http://pandas.pydata.org/pandas-docs/stable/api.html#dataframe>`_
+      - a dict where each key is a column
```

Comment:
- a dict which will be wrapped by a pandas DataFrame
include a link to pandas DataFrame